editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -5596,7 +5596,7 @@ void CanvasItemEditorViewport::_create_nodes(Node *parent, Node *child, String &
 	// make visible for certain node type
 	if (Object::cast_to<Control>(child)) {
 		Size2 texture_size = texture->get_size();
-		undo_redo->add_do_property(child, "rect_size", texture_size);
+		undo_redo->add_do_property(child, "size", texture_size);
```

Comment:
Confirmed that this was the last remaining occurrence of `"rect_size"` used as a string (and couldn't find other `"rect_.*"` strings outside of the 3-to-4 converter code).